lib/rules/no-tabs.js

```diff
@@ -21,25 +22,51 @@ module.exports = {
             category: "Stylistic Issues",
             recommended: false
         },
-        schema: []
+        schema: [],
+        fixable: "whitespace"
```

Comment:
Now this should be `"code"` instead of `"whitespace"` since this fixing modifies literals.